torch/lib/c10d/test/FileStoreTest.cpp

```diff
@@ -75,7 +75,7 @@ void testHelper(const std::string prefix = "") {
 
   // Check that the counter has the expected value
   {
-    c10d::FileStore fileStore(path);
+    c10d::FileStore fileStore(path, 1);
```

Comment:
Needs to be numThreads + 1, same as above, or the file will be gone before the validation can be done.